website/content/en/docs/setup/quickstart.md

```diff
@@ -1,14 +1,16 @@
 ---
-title: Vector quickstart
+title:       Vector quickstart
```

Comment:
This was auto-formatted. If it's too distracting, I can revert the formatting changes.